pkg/api/v1/conversion_generated.go

```diff
@@ -23,6 +23,7 @@ import (
 
 	api "k8s.io/kubernetes/pkg/api"
 	resource "k8s.io/kubernetes/pkg/api/resource"
+	experimental "k8s.io/kubernetes/pkg/apis/experimental"
```

Comment:
I still really don't like having this import, even temporarily. It implies that some generator is using the wrong version for something, no? Or has the wrong idea of what things it should generate for what version?